apps/site/lib/site_web/views/trip_plan_view.ex

```diff
@@ -586,6 +610,13 @@ defmodule SiteWeb.TripPlanView do
             conn.assigns.itinerary_row_lists,
             Stream.iterate(1, &(&1 + 1))
           ]) do
+      {itinerary_includes_sl1, i} =
+        if Enum.any?(it, fn leg -> Leg.is_sl1?(leg) end) do
```

Comment:
I think this logic might not be sufficient. We want the origin or destination trip to be on the SL, but I think the origin or destination needs to be the airport specifically. If I map a trip from South Station to the World Trade Center I don't think we want to show the airport messages.